docs/_documentations/debugging.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Debugging in Codewind
+description: Debugging in Codewind
+keywords: debug, mode, applications, application, project, menu, view, attaching, attach, actions, action, run, restart in debug mode, attach debugger, debug process, debugging process
+duration: 1 minute
+permalink: debugging
+type: document
+---
+
+# Debugging in Codewind
+
+Complete the following steps to enable debugging in Codewind applications:
+
+1. In the Codewind view, right-click a project and select **Restart in Debug Mode** from the menu.
+2. Wait for the project state to switch to **Debugging**. The first time that the project switches to **Debugging**, the debugger is attached.
+3. When in debug mode, you can perform the debugging actions that are included with your IDE.
```

Comment:
When in debug mode, you can debug your application as long as the debugger is attached to the debug process.  You can perform the debugging actions that are included and supported by the IDE that you are using. 

Note that for VS Code, some additional extensions must be installed first. For example, for Java applications, you have to install the Java language pack and a debugger.  For Eclipse, since Codewind requires the Java EE package, Java debug support is already provided.